diff --git a/dpdkbuild/Makefile b/dpdkbuild/Makefile
index 64da6cc32..56897cbe4 100644
--- a/dpdkbuild/Makefile
+++ b/dpdkbuild/Makefile
@@ -11,6 +11,8 @@ include $(SPDK_ROOT_DIR)/mk/spdk.common.mk
 
 DPDK_OPTS = -Denable_docs=false
 DPDK_OPTS += -Dtests=false
+DPDK_OPTS += -Dplatform=generic
+
 
 ifeq ($(shell uname -m),aarch64)
 ifeq ($(shell uname -p),unknown)
diff --git a/dpdk/config/x86/meson.build b/dpdk/config/x86/meson.build
index 8087b9ae91..a236a316dd 100644
--- a/dpdk/config/x86/meson.build
+++ b/dpdk/config/x86/meson.build
@@ -50,15 +50,6 @@ optional_flags = [
         'AES',
         'AVX',
         'AVX2',
-        'AVX512BW',
-        'AVX512CD',
-        'AVX512DQ',
-        'AVX512F',
-        'AVX512VL',
-        'PCLMUL',
-        'RDRND',
-        'RDSEED',
-        'VPCLMULQDQ',
 ]
 foreach f:optional_flags
     if cc.get_define('__@0@__'.format(f), args: machine_args) == '1'
